.release/buildspec_sign.yml

```diff
@@ -44,6 +44,21 @@ phases:
       - ls -lah $(dirname `head -1 ./$GIT_COMMIT_ID.manifest`)
   post_build:
     commands:
+      - echo "Copying artifacts to generate a latest binary"
+      - VERSION=`cat ./$GIT_COMMIT_ID.version`
+      - ARTIFACT_DIRECTORY=`cat ./$GIT_COMMIT_ID.artifactdir`
+      - |
+        for platform in windows darwin linux linux-arm64
+        do
+            extension=""
+            if [ "$PLATFORM" = "windows"]; then
+                extension=".exe"
+            fi
+            cp $ARTIFACT_DIRECTORY/copilot-$platform-$VERSION$extension $ARTIFACT_DIRECTORY/copilot-$platform$extension
+            cp $ARTIFACT_DIRECTORY/copilot-$platform-$VERSION$extension.asc $ARTIFACT_DIRECTORY/copilot-$platform$extension.asc
```

Comment:
```suggestion
            cp $ARTIFACT_DIRECTORY/copilot-$platform-$VERSION$extension.asc $ARTIFACT_DIRECTORY/copilot-$platform$extension.asc
            cp $ARTIFACT_DIRECTORY/copilot-$platform-$VERSION.md5 $ARTIFACT-DIRECTORY/copilot-$platform.md5
```

We need to also generate md5s for the latest versions as we currently release the checksums for all our artifacts. Alternatively, we can drop generating md5s altogether , but that's another conversation.

Reply:
OK updated the PR so that we don't do these steps and keep it simpler by moving it to the buildspec